mg.js

```diff
@@ -0,0 +1,17 @@
+const img = document.createElement("img");
+const cardButton = document.querySelector(".card");
+const backImg = "img/back.jpeg";
+const frontImg = "img/front.jpg";
+img.src = backImg;
+cardButton.appendChild(img);
+
+let isFlipped = false;
```

Comment:
Do you consider `isFlipped = false` to be displaying the front or the back of the image? There is confusion in that logic because I need to click twice on the card to see it change. It only happens for the first flip. Afterwards, the card reacts after a single click, as expected.

Changing the variable's name to `showFront` or `showBack` would help understand the logic.